webapp/channels/src/components/multiselect/multiselect.tsx

```diff
@@ -332,7 +332,7 @@ export default class MultiSelect<T extends Value> extends React.PureComponent<Pr
             noteTextContainer = (
                 <div className='multi-select__note'>
                     <div className='note__icon'>
-                        <LocalizedIcon
+                        <<i/>
                             className='fa fa-info'
                             title={{id: 'generic_icons.info', defaultMessage: 'Info Icon'}}
```

Comment:
we also need to add react-intl here, can you please take a look https://github.com/mattermost/mattermost/issues/25109#issue-1959027425

Reply:
I dont Exactly know this " add react-intl here"